README.md

```diff
@@ -1,6 +1,20 @@
 # ArcGIS Maps SDK for Swift Samples
 
-This repository contains Swift sample code demonstrating the capabilities of [ArcGIS Maps SDK for Swift](https://developers.arcgis.com/swift/) and how to use them in your own app. The project can be opened in Xcode and run on a simulator or a device. 
+This repository contains Swift sample code demonstrating the capabilities of the [ArcGIS Maps SDK for Swift](https://developers.arcgis.com/swift/) and how to use those capabilities in your own app. The project can be opened in Xcode and run on a simulator or a device.
+
+## Features
+
+* Maps - Open, create, interact with and save maps
+* Scenes - Visualize 3D environments and symbols
+* Layers - Display vector and raster data in maps and scenes
+* Augmented Reality - View data overlaid on the real world through your device's camera
+* Visualization - Show graphics, popups, callouts, sketches, and style maps with symbols and renderers
+* Edit and Manage Data - Add, delete, and edit features and attachments, and taking data offline
+* Search and Query - Find addresses, places, and points of interest
+* Routing and Logistics - Calculate routes between locations and around barriers
+* Analysis - Perform spatial analysis via geoprocessing tasks and services
+* Cloud and Portal - Search for webmaps and securely connect to your portal
```

Comment:
```suggestion
* Cloud and Portal - Search for web maps and securely connect to your portal
```

Is this supposed to be one word?

Reply:
web map indeed: https://support.esri.com/en-us/gis-dictionary/search?q=web+map